extensions/amp-story/1.0/amp-story-page-attachment.js

```diff
@@ -52,6 +54,15 @@ const AttachmentType = {
   OUTLINK: 1,
 };
 
+/**
+ * @enum {string}
+ */
+const FormResponseAttribute = {
```

Comment:
Should we import `amp-form.FormState` instead of duplicating it?

Reply:
I'm leaning towards not using `FormState` because it represents the 6 possible [states](https://amp.dev/documentation/components/amp-form/#classes-and-css-hooks) of the form, whereas the `FormResponseAttribute` represents the 3 attributes used for rendering responses in the form